scripts/drsinter_data_extraction.py

```diff
@@ -215,18 +253,19 @@ def isNaN(string):
 
 metadata.loc[0] = ['Data Sampling Rate (s)', sampling_rate]
 
-### Get channel names (with NaN for missing column entries)
+# Get channel names (with NaN for missing column entries)
 channel_row = raw_data[0].loc[raw_data[0][dummy_names[0]] == 'Waveform Name']
 
 header_print('DATA UNITS')
 
-### Get units for each channel (TODO: remove weird non-standard character in front of C for temp)
+# Get units for each channel (TODO: remove weird non-standard character in front of C for temp)
```

Comment:
```suggestion
# Get units for each channel
```

I must have neglected to remove this TODO when I added removal of non-ASCII characters. If you agree, we should probably remove part of this comment.

Reply:
added into second commit